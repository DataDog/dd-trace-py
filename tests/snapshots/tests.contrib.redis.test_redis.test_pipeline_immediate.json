[[
  {
    "name": "redis.command",
    "service": "redis",
    "resource": "SET a 1",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "redis",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "component": "redis",
      "db.system": "redis",
      "language": "python",
      "out.host": "localhost",
      "redis.raw_command": "SET a 1",
      "runtime-id": "3cf1df7fb079462ab81608355e026651",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "network.destination.port": 6379,
      "out.redis_db": 0,
      "process_id": 19999,
      "redis.args_length": 3
    },
    "duration": 343500,
    "start": 1692651821823333917
  }],
[
  {
    "name": "redis.command",
    "service": "redis",
    "resource": "SET a 1",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "redis",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "component": "redis",
      "db.system": "redis",
      "language": "python",
      "out.host": "localhost",
      "redis.raw_command": "SET a 1",
      "runtime-id": "3cf1df7fb079462ab81608355e026651",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "network.destination.port": 6379,
      "out.redis_db": 0,
      "process_id": 19999,
      "redis.pipeline_length": 1
    },
    "duration": 158750,
    "start": 1692651821823756750
  }]]
