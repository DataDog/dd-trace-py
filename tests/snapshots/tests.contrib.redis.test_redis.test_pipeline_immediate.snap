[[{"name" "redis.command"
   "service" "redis"
   "resource" "SET a 1"
   "type" "redis"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1596062188720270000
   "duration" 2954000
   "meta" {"runtime-id" "601815e8709646a9ac897c59376c80b6"
           "redis.raw_command" "SET a 1"
           "out.host" "localhost"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_dd.tracer_kr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 76411
              "out.port" 6379
              "out.redis_db" 0
              "redis.args_length" 3}}]
 [{"name" "redis.command"
   "service" "redis"
   "resource" "SET a 1"
   "type" "redis"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1596062188723313000
   "duration" 818000
   "meta" {"runtime-id" "601815e8709646a9ac897c59376c80b6"
           "redis.raw_command" "SET a 1"
           "out.host" "localhost"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_dd.tracer_kr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 76411
              "out.port" 6379
              "out.redis_db" 0
              "redis.pipeline_length" 1}}]]
